src/test/java/tests/HowItWorksTest.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.HowItWorksPage;
+
+public class HowItWorksTest extends BaseTest {
+
+    HomePage homePage = new HomePage(driver);
+    HowItWorksPage page = new HowItWorksPage(driver);
+
+    @BeforeTest
+    public void howItWorksPage(){
+        homePage.openHomePage()
+                .goToHowItWorks();
+    }
+
+    @Test
+    public void getPageTitle(){
```

Comment:
Test methods name should discribe what we have to do and what we have to assert, so please rename tests method name. Also your test should follow "AAA" pattern (here the link: " https://cypriops.com/blog/test-food-chain/ " or " https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80 ").